core/src/reporter/reporter.ts

```diff
@@ -61,6 +69,24 @@ export function reporter(state: State, logger: Logger) {
         }
       }
     }
+
+    // check if payload uses Submit method, which indicates submission if for Aggregator Price Feeds
```

Comment:
This is not optimal way to solve this problem. It would be much better to create a "precall" and "postcall" functions that will be executed when defined for a certain reporter types as we have discussed before. We should not include this code to general reporter function. @bayram98 Please make an issue for removing it from here.